app/src/main/java/org/p2p/wallet/sdk/facade/SolendSdkFacade.kt

```diff
@@ -0,0 +1,211 @@
+package org.p2p.wallet.sdk.facade
+
+import com.google.gson.Gson
+import org.p2p.solanaj.rpc.NetworkEnvironment
+import org.p2p.wallet.home.model.Token
+import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
+import org.p2p.wallet.infrastructure.network.environment.NetworkEnvironmentManager
+import org.p2p.wallet.sdk.SolendSdk
+import org.p2p.wallet.sdk.facade.mapper.SolendMethodResultMapper
+import org.p2p.wallet.sdk.facade.model.SolendConfigResponse
+import org.p2p.wallet.sdk.facade.model.SolendEnvironment
+import org.p2p.wallet.sdk.facade.model.SolendFeePayerTokenData
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendCollateralAccount
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendCollateralAccountsList
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendDepositTransactions
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendMarketInformation
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendTokenDepositFees
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendUserDeposit
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendUserDepositByTokenResponse
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendUserDepositsResponse
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess.SolendWithdrawTransactions
+import org.p2p.wallet.utils.Base58String
+import timber.log.Timber
+import kotlinx.coroutines.withContext
+
+private const val TAG = "SolendSdkFacade"
+
+class SolendSdkFacade(
+    private val solendSdk: SolendSdk,
+    private val solendEnvironment: SolendEnvironment,
+    private val networkEnvironmentManager: NetworkEnvironmentManager,
+    private val methodResultHandler: SolendMethodResultHandler,
+    private val methodResultMapper: SolendMethodResultMapper,
+    private val gson: Gson,
+    private val dispatchers: CoroutineDispatchers
+) {
+
+    private val currentNetworkEnvironment: NetworkEnvironment
+        get() = networkEnvironmentManager.loadCurrentEnvironment()
+
+    suspend fun createDepositTransactions(
+        relayProgramId: String,
+        depositAmount: ULong,
+        currencySymbol: String,
+        ownerAddress: Base58String,
+        lendingMarketAddress: String?,
+        currentBlockhash: String,
+        remainingFreeTransactionsCount: UInt,
+        payFeeWithRelay: Boolean,
+        feePayerToken: SolendFeePayerTokenData?,
+        realFeePayerAddress: Base58String
+    ) {
+        withContext(dispatchers.io) {
+            Timber.tag(TAG).i("Method createSolendDepositTransactions called")
```

Comment:
I think these logs are redundant. 
There are any blockers and this code will be reached anyway

Reply:
we need to trace which method of SolendSdk was called when diagnosing errors in Sentry